src/LayoutComponents/nav/SideNav.jsx

```diff
@@ -103,15 +103,15 @@ class SideNav extends Component {
     const { expandedState, pages, parents } = this.props;
     const panels = renderPanels(parents, pages);
     return (
-      <div className='sideNav' id='side-nav'>
-        <PanelGroup>
+      <nav className='sideNav' id='side-nav'>
```

Comment:
I could have sworn I had this as a `<nav>` element. This is shameful on my part...